### PR TITLE
Backport PR #1118 to 8.x and adapt list of forms

### PR DESCRIPTION
**Pull request by matks**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 8.x
| Description?  | Backport PR 1118 to 8.x and adapt list of forms

### CODE REVIEW

File: development/components/form/form-theme/form-theme.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+- form_div_layout
+- typeahead
+- material
+
+By default, PrestaShop's Twig templates will use `bootstrap_4_horizontal_layout.html.twig` (as configured in PrestaShop's `/app/config/config.yml` file). This child of the above theme render forms horizontally instead of vertically.
+
+{{< figure src="../img/old-form-theme-17.png" title="Original Form Theme example of rendering" >}}
+
+### PrestaShop UI Kit Form theme
+{{< minver v="1.7.7" title="true" >}}
```
Comment (eternoendless) [inline development/components/form/form-theme/form-theme.md]: This part probably needs to be updated

File: development/components/form/form-theme/form-theme.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+```
+{% form_theme form 'PrestaShopBundle:Admin/TwigTemplateForm:prestashop_ui_kit_base.html.twig' %}
+```
+
+{{< figure src="../img/ui-kit-form-theme.png" title="UI Kit Form Theme example of rendering" >}}
+
+
+
+[sf-bootstrap4-form-theme]: https://symfony.com/doc/4.4/form/bootstrap4.html
+[ui-kit]: {{< ref "/1.7/development/uikit.md" >}}
```
Comment (eternoendless) [inline development/components/form/form-theme/form-theme.md]:
```suggestion
[ui-kit]: {{< ref "/8/development/uikit.md" >}}
```

File: development/components/form/form-theme/form-theme.md (hunk 1 above)
Comment (matks) [reply development/components/form/form-theme/form-theme.md]: Done in https://github.com/PrestaShop/docs/pull/1175/commits/e05b19e16810d446a92fe897e052c0d912a0897f

**Comment by matks** [comment]
Ready for new review 👍

**Review by PierreRambaud — APPROVED** [review]

**Review by Progi1984 — APPROVED** [review]

**Comment by Progi1984** [comment]
Thanks @matks

**Pull request merged**